microsoft-dynamics-365/2.3/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -110,6 +110,77 @@ Release Notes: xref:release-notes::connector/microsoft-dynamics-365-connector-re
 * <<on-modified-object>>
 * <<on-new-object>>
 
+[NOTE]
+Microsoft Dynamics 365 Connector sources (listeners) use Object Store to save watermarks. You might experience limitations that are specific to the Object Store implementation you are using (Object Store for CloudHub deployments or Object Store for on-premises deployments), so configure Object Store to suit your needs. +
```

Comment:
Another option:
```suggestion
Microsoft Dynamics 365 Connector sources (listeners) use Object Store to save watermarks. To avoid encountering limitations, configure Object Store implementation that best suits your environment: Object Store for CloudHub deployments or Object Store for on-premises deployments. +
```